src/pages/AddProperty.tsx

```diff
@@ -13,14 +14,20 @@ interface DummyProperty {
     state_province?: string | null
 }
 
-const AddProperty = () => {
-    let navigate = useNavigate();
+interface AddPropertyProps {
+    goBack: () => void;
+}
+
+const AddProperty = (props: AddPropertyProps) => {
```

Comment:
If this component functions as something other than the component to add a property, please rename it (and it's file) to something appropriate.